README.MD

```diff
@@ -76,7 +76,7 @@ v.bootstrap();
 ```powershell
 git clone https://github.com/haacked/aspnet-client-validation.git
 npm install
-script/build   # If using PowerShell: script/build.ps1
+script/scripts/build   # If using PowerShell: script/scripts/build.ps1
```

Comment:
I think the `scripts` folder within `script` is a mistake on my part. 😆  I'll probably fix that later. This change is fine for now.